docs/image-config/README.md

```diff
@@ -0,0 +1,142 @@
+
+The location of the mounted ConfigMap is then referenced in the [`OperatorConfig`](https://github.com/k8ssandra/cass-operator/blob/master/config/manager/controller_manager_config.yaml#L1).
+
+It is important to note that all image configurations occur in the ConfigMap mounted into the cass-operator deployment, not via the K8ssandraCluster or CassandraDatacenter CRs. Even though the `ImageConfig` looks like a Kubernetes API extension (and even has a `Kind`, and `ApiVersion`), **it is not**.
+
+To specify a custom image registry, we recommend you create a kustomization which may look something like the following:
+
+```
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - config/deployments/cluster
+patches:
+- target:
+    kind: ConfigMap
+    name: cass-operator-manager-config
+  patch: |
+    - op: replace 
+      path: /data/image_config.yaml
+      value: |
+        # Your custom config goes here. Please refer to `./config/manager/image_config.yaml` for an example
+        # of what fields and configurations are available.
+```
+
+**Note: there are some complexities in changing these configurations at runtime. In particular:**
+1. **The ImageConfig ConfigMap will be automatically updated in the pod filesystem  when it changes, but changes will not take effect until the Operator is restarted.**
+2. **If a CassandraDatacenter is already running, the new image configurations will not be automatically applied. The CassandraDatacenter needs to be deleted and re-created in order for the changes to take effect.**
+
+## Stargate and Reaper images
+
+The images for Reaper, Stargate, the short lived JMX container within the Cassandra pods, and the Cassandra/management-api containers can be also set via the K8ssandraCluster CR as follows:
+
+```
+apiVersion: k8ssandra.io/v1alpha1
+kind: K8ssandraCluster
+metadata:
+  name: test
+spec:
+  jmxInitContainerImage: 
+    registry: <my-registry>
+    repository: <my-repository>
+    image: <my-image>
+    name: <my-name>
+    tag: <my-tag>
+    pullPolicy: <my-pullPolicy>
+    pullSecretRef: <my-pullSecretRef>
+  serverImage: <<custom-registry/my-jmx-container-image:custom-tag>>
+  medusa:
+    containerImage:
+      registry: <my-registry>
+      repository: <my-repository>
+      image: <my-image>
+      name: <my-name>
+      tag: <my-tag>
+      pullPolicy: <my-pullPolicy>
+      pullSecretRef: <my-pullSecretRef>
+  stargate:
+    size: 1
+    containerImage:
+      registry: <my-registry>
+      repository: <my-repository>
+      image: <my-image>
+      name: <my-name>
+      tag: <my-tag>
+      pullPolicy: <my-pullPolicy>
+      pullSecretRef: <my-pullSecretRef>
+          stargate:
+  reaper:
+    containerImage:
+      registry: <my-registry>
+      repository: <my-repository>
+      image: <my-image>
+      name: <my-name>
+      tag: <my-tag>
+      pullPolicy: <my-pullPolicy>
+      pullSecretRef: <my-pullSecretRef>
+  cassandra:
+    serverVersion: "3.11.11"
```

Comment:
We should stop using 3.11.11 as example. Just migrate examples to 4.0.x

Reply:
OK.